app/views/unattended/provisioning_templates/snippet/pxegrub2_discovery.erb

```diff
@@ -3,13 +3,13 @@ kind: snippet
 name: pxegrub2_discovery
 model: ProvisioningTemplate
 snippet: true
--%>
-<%#
-RHEL 7 virtio driver does not have persistent naming scheme, causing interfaces
-to be named eth0, eth1, etc.. If you want to enable the new persistent naming
-scheme and get inteface names like ens3, add net.ifnames=1 to the linuxefi line
-below. This will not be fixed until RHEL8 due to prevent breaking changes for
-existing systems. See RHBZ#1259015 for more details.
+description: |
+  TODO
```

Comment:
@lzap could you please provide a suggestion? The text below does not sound like a full description.

Reply:
Snippet with grub2 menu items for discovery plugin. It is included in PXEGrub2 Default templates to boot unknown hosts into discovery image.